libs/http-parser/Makefile

```diff
@@ -7,30 +7,27 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=libhttp-parser
```

Comment:
+1 in non renaming. Otherwise please go through the packages and replace with the new name.